From 14a42898c928d670fa0d691369a0ac9e9e1f4dc2 Mon Sep 17 00:00:00 2001
From: Yuchou Gan <yuchou.gan@nxp.com>
Date: Fri, 30 Mar 2018 17:41:19 +0800
Subject: [PATCH 3677/5242] MLK-18101-1 include: soc: imx8: sc: types: Add
 SC_C_SEL0 for B0 imx8qxp board

commit  49d1822009738c5aa32b5064537b7335aa4b9ef6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add SC_C_SEL0 for imx8qm/qxp B0.

Signed-off-by: yuchou gan <yuchou.gan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/soc/imx8/sc/types.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/soc/imx8/sc/types.h b/include/soc/imx8/sc/types.h
index 4192809..3309e03 100644
--- a/include/soc/imx8/sc/types.h
+++ b/include/soc/imx8/sc/types.h
@@ -762,6 +762,7 @@
 	SC_C_OFS_IRQ = 42,
 	SC_C_RST0 = 43,
 	SC_C_RST1 = 44,
+	SC_C_SEL0 = 45,
 	SC_C_LAST
 } sc_ctrl_t;
 
-- 
1.7.9.5

